arithmetization/src/main/java/net/consensys/linea/zktracer/container/stacked/ModuleOperationStackedSet.java

```diff
@@ -38,15 +39,17 @@
  */
 @Accessors(fluent = true)
 public class ModuleOperationStackedSet<E extends ModuleOperation> extends StackedSet<E> {
+  private static final int EXPECTED_PROPORTION_OF_DUPLICATE = 1; // TODO: find me !
   private static final Logger log = LoggerFactory.getLogger(ModuleOperationStackedSet.class);
   private final CountOnlyOperation lineCounter = new CountOnlyOperation();
   @Getter private boolean conflationFinished = false;
+  private boolean duplicateAlreadyRemoved = false; // TODO: add documentation about this
 
   public ModuleOperationStackedSet() {
     super();
   }
 
-  /** Prefer this constructor as we preallocate more needed memory */
+  /** Prefer this constructor as we preallocate more necessary memory */
   public ModuleOperationStackedSet(
```

Comment:
noticed that this is not used at all even though it's mentioned that it is the preferred

Reply:
Yes, we need real block values to know which size we need to inialize.

Reply:
https://github.com/Consensys/linea-tracer/issues/1394